src-java/base-topology/base-storm-topology/src/main/java/org/openkilda/wfm/share/service/SpeakerFlowSegmentRequestBuilder.java

```diff
@@ -158,8 +168,16 @@ public List<FlowSegmentRequestFactory> buildIngressOnly(
     public List<FlowSegmentRequestFactory> buildIngressOnly(
             CommandContext context, Flow flow, FlowPath path, FlowPath oppositePath,
             SpeakerRequestBuildContext speakerRequestBuildContext, MirrorContext mirrorContext) {
+        return mergePair(makeRequests(context, flow, path, oppositePath, true, false, false,
+                speakerRequestBuildContext, mirrorContext));
+    }
+
+    @Override
+    public Pair<FlowSegmentRequestFactoriesSequence, FlowSegmentRequestFactoriesSequence> buildIngressOnlySeverally(
```

Comment:
Caller pass 2 flow paths in argument. The result pair contain results for `path` (left) and for `oppositePath` (right). Left/right corresponding to order of passed paths arguments.

The only better response option is own object instead of already existed `Pair`. Should I make it and improve this result verbosity?

Reply:
Could you mention this in the method's javadoc?